boot/freeldr/freeldr/lib/peloader.c

```diff
@@ -518,6 +520,29 @@ PeLdrpScanImportAddressTable(
 
 /* FUNCTIONS *****************************************************************/
 
+BOOLEAN
+PeLdrInitializeModuleList(VOID)
+{
+    PLDR_DATA_TABLE_ENTRY FreeldrDTE;
+
+    InitializeListHead(&FrLdrModuleList);
+
+    /* Allocate a data table entry for freeldr.sys.
+       The base name is scsiport.sys for imports from ntbootdd.sys */
+    if (!PeLdrAllocateDataTableEntry(&FrLdrModuleList,
+                                     "scsiport.sys",
+                                     "freeldr.sys",
+                                     &__ImageBase,
+                                     &FreeldrDTE))
+    {
+        /* Cleanup and bail out */
+        ERR("Failed to allocate DTE for freeldr\n");
+        return FALSE;
+    }
```

Comment:
There is currently nothing that imports from "freeldr.sys", so we don't need an export entry for "freeldr.sys". This will change later, when rosload.exe will import from "freeldr.sys", and ntbootdd.sys imports to "scsiport.sys" will be resolved by rosload.exe.